src/user/UserRouter.ts

```diff
@@ -1,50 +1,32 @@
-import { Request, Response, NextFunction, Router } from 'express';
+import { Request, Response, Router } from 'express';
 import UserService from './UserService';
+import { check, validationResult } from 'express-validator';
 
 const router = Router();
 
-const validateUsername = (req: Request, res: Response, next: NextFunction) => {
-  //
+router.post(
+  '/api/1.0/users',
+  check('username').notEmpty().withMessage('Username cannot be null'),
```

Comment:
Using express-validator to validate the request body fields instead of creating a manual middleware for validaton